Packs/CommonTypes/ReleaseNotes/3_3_52.md

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
I don't understand this.  
Are all the incidents fields new or is Source ID the only new one and the rest have been updated?
If they are all new, then put the word New in front of them.
If they are updated, then put the word Updated in front of them.

Reply:
The rest were just updated, so it's automatically generated from the sdk, in any case I'll add updated